app/src/main/java/org/p2p/wallet/sdk/facade/SolendSdkFacade.kt

```diff
@@ -34,7 +33,8 @@ class SolendSdkFacade(
 ) {
 
     private val currentNetworkEnvironment: NetworkEnvironment
-        get() = networkEnvironmentManager.loadCurrentEnvironment()
+        //        get() = networkEnvironmentManager.loadCurrentEnvironment()
+        get() = NetworkEnvironment.MAINNET
```

Comment:
MAINNET is fixed rn?

Reply:
this is the only one which works for now